Clover/app/src/main/java/org/floens/chan/core/presenter/ThreadPresenter.java

```diff
@@ -713,9 +720,18 @@ private void showPostInfo(Post post) {
             text.append("\nTripcode: ").append(post.tripcode);
         }
 
-        /*if (!TextUtils.isEmpty(post.countryName)) {
-            text += "\nCountry: " + post.country + ", " + post.countryName;
-        }*/
+        if (post.httpIcons != null && !post.httpIcons.isEmpty()) {
+            for (PostHttpIcon icon : post.httpIcons) {
+                if (icon.url.toString().contains("troll")) {
+                    text.append("\nTroll Country: ").append(icon.name);
+                } else if (icon.url.toString().contains("country")) {
+                    text.append("\nCountry: ").append(icon.name);
+                } else {
```

Comment:
might be good to explicitly call out the since4pass icon type instead of using an else. in case they ever add another icon, we won't show the since4pass instead

Reply:
Yep, I agree.